ChaosMod/Memory/Hooks/AudioClearnessHook.cpp

```diff
@@ -0,0 +1,41 @@
+#include <stdafx.h>
+
+#include "Hook.h"
+
+static bool ms_bEnabledHook = false;
+static int ms_fValue = 0;
+
+__int64(*_OG_rage__audRequestedSettings__SetClearness)(__int64 _this, __int64 pitch);
+__int64 _HK_rage__audRequestedSettings__SetClearness(__int64 _this, __int64 pitch)
+{
+	return _OG_rage__audRequestedSettings__SetClearness(_this, ms_bEnabledHook ? ms_fValue : pitch);
+}
```

Comment:
```suggestion
__int64(*_OG_rage__audRequestedSettings__SetClearness)(__int64 _this, __int64 clearness);
__int64 _HK_rage__audRequestedSettings__SetClearness(__int64 _this, __int64 clearness)
{
	return _OG_rage__audRequestedSettings__SetClearness(_this, ms_bEnabledHook ? ms_fValue : clearness);
}
```
Not sure pitch would fit in this scenario?

Reply:
sloppy copy paste LOL